src/Mod/Arch/exportIFC.py

```diff
@@ -54,10 +54,6 @@
 __author__ = ("Yorik van Havre", "Jonathan Wiedemann", "Bernd Hahnebach")
 __url__    = "https://www.freecad.org"
 
-# Save the Python open function because it will be redefined
-if open.__module__ in ['__builtin__', 'io']:
-    pyopen = open
-
```

Comment:
the new import replacing this is missing

Reply:
Thanks. I pushed a fix. The import was missing from `importGBXML.py`